mmf/modules/detr/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
As discussed earlier let's not have a `detr` module under `modules` directory. Dataset specific files can go under `datasets` and util files can go under `utils` directory respectively.

Reply:
(this has been resolved in the update; now there is no `detr` module and the tools are now in `mmf/utils/box_ops.py` and `mmf/utils/image_transforms.py`)